bindb.go

```diff
@@ -2,15 +2,15 @@ package bindb
 
 import (
 	"bufio"
+	"errors"
 	"fmt"
 	"github.com/iancoleman/strcase"
 	"os"
 	"strconv"
 	"strings"
 )
 
-type BINDBrecord struct {
-	BIN     string
+type BINDBRecord struct {
```

Comment:
I would just name this struct as "Record" since it's already in bindb package (bindb.Record)